### PR TITLE
Update multiqc to v1.8

### PR DESCRIPTION
**Pull request by skrakau**
Update MultiQC to v1.8 to avoid bug when adding multiple FastQC sections, i.e. before and after trimming or contaminant removal.

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/mag branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/mag)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by skrakau** [comment]
Nice, yes definitely! :)

**Comment by apeltzer** [comment]
image

File: environment.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies:
   - conda-forge::pymdown-extensions=6.0
   - conda-forge::pygments=2.5.2
   - bioconda::fastqc=0.11.8
-  - bioconda::multiqc=1.7
+  - bioconda::multiqc=1.8
```
Comment (apeltzer) [inline environment.yml:14]:
```suggestion
  - bioconda::multiqc=1.9
```

File: environment.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ dependencies:
...
   - conda-forge::matplotlib=3.1.1
```
Comment (apeltzer) [inline environment.yml:15]:
```suggestion
```

Comment (apeltzer) [reply environment.yml:15]: Outdated, can be dropped as MultiQC v1.9 uses proper matplotlib now :-)

**Pull request merged**